test/src/test_cases/edx-schools-frontend/test-school-inbox.js

```diff
@@ -23,6 +23,7 @@ fixture `school-inbox`
        getToken().then(async (data) => {
             token = data.access_token;
             testExchange = await createSecureExchange(token, JSON.stringify(testExchange));
+            log.info("Secure exchange created with id: " + testExchange.secureExchangeID);
```

Comment:
Please remove

Reply:
Ummm weird. This code is not on my branch, not sure how it appearing here? https://github.com/bcgov/EDUC-EDX/blob/feature/edx-10/test/src/test_cases/edx-schools-frontend/test-school-inbox.js